glib/src/object.rs

```diff
@@ -1115,6 +1119,10 @@ impl Object {
             .unwrap())
     }
 
+    /// Create a new instance of an object of the given type with the given properties.
+    ///
+    /// This fails if the object is not instantiable, doesn't have all the given properties or
+    /// property values of the wrong type are provided.
     pub fn with_type(
```

Comment:
I wonder if there are situations where you want to use `with_type`  instead of `new` or if `new_type` should rather be private.

Reply:
Yes, if all you have is the `GType`. I need this in some places :)

Reply:
Think of factories, for example.